src/main/java/com/echobox/api/linkedin/connection/OrganizationConnection.java

```diff
@@ -0,0 +1,407 @@
+import java.util.List;
+
+/**
+ * Organization connection class that should contain all organization operations
+ * 
+ * @author joanna
+ *
+ */
+public class OrganizationConnection extends ConnectionBase {
+
+  private static final String ORGANIZATIONAL_ENTITY_ACLS = "/organizationalEntityAcls";
+  private static final String ORGANIZATIONS = "/organizations";
+  private static final String ORGANIZATIONAL_ENTITY_FOLOWER_STATS =
+      "/organizationalEntityFollowerStatistics";
+  private static final String ORGANIZATIONAL_PAGE_STATS = "/organizationPageStatistics";
+  private static final String ORGANIZATION_ENTITY_SHARE_STATS =
+      "/organizationalEntityShareStatistics";
+  
+  private static final String QUERY_KEY = "q";
+  private static final String ROLE_KEY = "role";
+  private static final String STATE_KEY = "state";
+  private static final String ORGANIZATIONAL_TARGET_KEY = "organizationalTarget";
+  private static final String VANITY_NAME_KEY = "vanityName";
+  private static final String EMAIL_DOMAIN_KEY = "emailDomain";
+  private static final String ORGANIZATIONAL_ENTITY_KEY = "organizationalEntity";
+  private static final String ORGANIZATION_KEY = "organization";
+  private static final String SHARES_KEY = "shares";
+  
+  private static final String ROLE_ASSIGNEE_VALUE = "roleAssignee";
+  private static final String ORGANIZATION_TARGET_VALUE = "organizationalTarget";
+  private static final String VANITY_NAME_VALUE = "vanityName";
+  private static final String EMAIL_DOMAIN_VALUE = "emailDomain";
+  private static final String ORGANIZATIONAL_ENTITY_VALUE = "organizationalEntity";
+  private static final String ORGANIZATION_VALUE = "organization";
+
+  /**
+   * Initialise an organization connection
+   * @param linkedinClient the linkedIn API client to create a LinkedIn organization connection
+   */
+  public OrganizationConnection(LinkedInClient linkedinClient) {
+    super(linkedinClient);
+    if (!Version.V2.equals(linkedinClient.getVersion())) {
+      throw new IllegalStateException(
+          "The LinkedIn clinet should be set to V2 to access the endpoints");
+    }
+  }
+
+  /**
+   * Find a Member's Organization Access Control Information
+   * @see <a href="https://docs.microsoft.com/en-us/linkedin/marketing/integrations/community-
+   * management/organizations/organization-access-control">Access Control</a>
+   * E.g. https://api.linkedin.com/v2/organizationalEntityAcls?q=roleAssignee
+   * @param role Limit results to specific roles, such as ADMINISTRATOR.
+   * @param state Limit results to specific role states, such as APPROVED.
+   * @param projection Field projection
+   * @return List of access controls for a given role and state for the member
+   */
+  public List<AccessControl> fetchMemeberOrganizationAccessControl(String role, String state,
+      Parameter projection) {
+    Parameter queryParam = Parameter.with(QUERY_KEY, ROLE_ASSIGNEE_VALUE);
+    Parameter roleParam = Parameter.with(ROLE_KEY, role);
```

Comment:
If role is null we will get an exception. Role and state are not required fields and if null maybe we should not include them as you do below